style/modules/footer.css

```diff
@@ -0,0 +1,66 @@
+.footer {
+    display: flex;
+    flex-wrap: nowrap;
```

Comment:
`nowrap` is the default, so it can come out (also the one under `.footer--midSection`)